File: Exercise/Medium/09. Pangram Checker/Code/Pangram Checker.java
import java.util.Scanner;

public class Main {
    public static boolean isPangram(String str) {
        boolean[] alphabet = new boolean[26];

        for (char ch : str.toLowerCase().toCharArray()) {
            if (Character.isLetter(ch)) {
                alphabet[ch - 'a'] = true;
            }
        }

        for (boolean present : alphabet) {
            if (!present) {
                return false;  
            }
        }
        return true;  
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        String str = scanner.nextLine();

        if (isPangram(str)) {
            System.out.println("True");
        } else {
            System.out.println("False");
        }

        scanner.close();
    }
}

File: Exercise/Medium/09. Pangram Checker/Code/Pangram Checker.cs
using System;
using System.Linq;

class Program {
    static bool IsPangram(string str) {
        bool[] alphabet = new bool[26];

        foreach (char ch in str.ToLower()) {
            if (char.IsLetter(ch)) {
                alphabet[ch - 'a'] = true;
            }
        }

        return alphabet.All(present => present);
    }

    static void Main() {
        string str = Console.ReadLine();

        if (IsPangram(str)) {
            Console.WriteLine("True");
        }
        else {
            Console.WriteLine("False");
        }
    }
}


